### ISSUE TITLE
blank export windows, not working  on linux

### ISSUE THREAD

**Issue by mvandrei** [Phildo/expandpass#5]
Hi,

There are two issues I'm facing:
1. On windows the output will be empty: expandpass -i seed.txt -o whatever.txt.
2. On Linux it will not run the app after being complied. It says expandpass: command not found.

What am I doing wrong?
Thanks in advance!

**Comment by Phildo** [comment]
1. how are you compiling on windows? I just pulled latest, and ran:
```
cl gen.cpp -o expandpass.exe
expandpass -i seed.txt -o whatever.txt
```
and it correctly populated `whatever.txt` with the expanded output from the example seed in `seed.txt` that comes with the repository.

2. "`expandpass: command not found`" is not an error from expandpass- it's an error from your environment (shell?). how are you compiling it? how are you trying to run it?

you'll need to give way more details if you want effective support.

the bottom line is, you'll need to compile it (`cl gen.cpp -o expandpass.exe` on windows, `cl gen.cpp -o expandpass` on everything else, where `cl` is whatever c++ compiler you want), and then run it from a command prompt by either putting the compiled executable somewhere on your `PATH`  or in your current working directory.

**Comment by mvandrei** [comment]
1. Yes, that's exactly how I did it, and the whatever.txt is empty. I've changed the output name, it generates the file, but without any content.
2. that is an error received in ubuntu using cl gen.cpp -o expandpass. It does the compiling, but when trying to use expandpass -i seed.txt -o whatever.txt it will give this error:
"~/Downloads/expandpass-master$ expandpass -i seed.txt -o whatever.txt
expandpass: command not found"

**Comment by Phildo** [comment]
I think you haven't extracted the zip.
If you're getting the repository by downloading the zip, you have to extract it before it can be used. It's weird that it's "successfully" compiling it within the zip in the first place.

If there are further problems, please just post the entire output of your shell.
